lnst/Recipes/ENRT/VlansRecipe.py

```diff
@@ -53,6 +53,15 @@ class VlansRecipe(VlanPingEvaluatorMixin,
         dict(gro="on", gso="on", tso="off", tx="off", rx="on"),
         dict(gro="on", gso="on", tso="on", tx="on", rx="off")))
 
+    vlan0_ipv4 = IPv4NetworkParam(default="192.168.10.0/24")
```

Comment:
hey do people think we should have a standard name for the IPv4Network param objects. 

For example if we values like `net1_ipv4`, `net2_ipv4, etc. I might be able to add `net1_ipv4`,net1_ipv6` to something like BaseEnrt then for test that require more they can add more to their own class or super class.

Reply:
i think this is specific enough to each test scenario that it is on the same level as the definition of the "network requirements"

so my opinion is that we should define the network parameters in the same class where we define `HostReq` and `DeviceReq` instances.

This basically means that we need to update each recipe individually and include the network parameters definition to each of them. Which is a bit of work, but I think it makes it very clear what and how the specific recipe and it's topology will look like/be configured.